Makefile

```diff
@@ -17,7 +16,7 @@ node_modules:
 	npm install
 
 types: node_modules
-	pdm run npx --no-install pyright tests $(PROJECT)
```

Comment:
I'm fine with this approach, but it may be better to start putting this config in `pyproject.toml`

Reply:
Good thought. I'm going to merge for now, but let's investigate this as a future improvement.